### PR TITLE
Mention python setup.py sdist

### PR DESCRIPTION
**Pull request by jean-philippe-martin**
I was stuck at that point, so thought I'd suggest a documentation update to help others who may run into the same issue.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/535/

Jenkins built the site at commit id 5f3115a185ab16678b8764440abdf2c3f499ac6b with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/260/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by aaltay** [comment]
R: @melap 

@jean-philippe-martin thank you for this PR. Could you make the change in the related `.md` file? `.html` files are auto-generated. And could you link to (https://docs.python.org/2/distutils/sourcedist.html) for more information.

**Comment by jean-philippe-martin** [comment]
Thank you @aaltay. I added the link; hopefully this time I found the correct file :)

File: src/documentation/sdks/python-pipeline-dependencies.md [hunk 1]
```diff
@@ -49,6 +49,12 @@ If your pipeline uses packages that are not available publicly (e.g. packages th
 
         --extra_package /path/to/package/package-name
 
+   Where package-name is the package's tarball. If you have the `setup.py` for that
```
Comment (melap) [inline src/documentation/sdks/python-pipeline-dependencies.md]: recommend the 'w' in Where to be lowercase

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/536/

Jenkins built the site at commit id 8817007cd67a182e85496d72c3b101646838b15f with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/260/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

File: src/documentation/sdks/python-pipeline-dependencies.md (hunk 1 above)
Comment (jean-philippe-martin) [inline src/documentation/sdks/python-pipeline-dependencies.md]: Done!

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/537/

Jenkins built the site at commit id 7becdfccbb115b10f9172137a2efb0dcb6f511b0 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/260/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

**Comment by aaltay** [comment]
Thank you!

**Pull request closed without merging**